packages/playground/src/components/select_node.vue

```diff
@@ -206,7 +209,10 @@ watch([loadingNodes, shouldBeUpdated], async ([l, s]) => {
       availableNodes.value = [];
       return;
     }
-    loadNodes(farmId);
+    clearTimeout(timeoutId.value);
+    timeoutId.value = setTimeout(() => {
+      loadNodes(farmId);
+    }, 1000);
```

Comment:
I think 1 second is a bit low, I don't think it will work on mainnet. can we increase the timeout? (maybe 5 seconds)

Reply:
this is a delay to start loading not a timeout on the loading itself. so ignore this comment